tests/xbot/common/injection/MockPermanentStorageWithDatabase.java

```diff
@@ -0,0 +1,13 @@
+package xbot.common.injection;
+
+import xbot.common.properties.PermanentStorageBase;
+
+public class MockPermanentStorageWithDatabase extends PermanentStorageBase {
+
+	public static String testFolder = "./488Database";
```

Comment:
This particular string is public, just in case a test needs to look directly at the database folder for some reason (or maybe clean it out later)